### PR TITLE
pidstore: remove invenio-records-rest from api app

### PR DESCRIPTION
**Pull request by jmartinm**
Signed-off-by: Javier Martin Montull <javier.martin.montull@cern.ch>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -224,8 +224,7 @@
             'crossref = inspirehep.modules.crossref:CrossRef',
             'inspire_orcid = inspirehep.modules.orcid:INSPIREOrcid',
             'inspire_disambiguation = inspirehep.modules.disambiguation:InspireDisambiguation',
-            'inspire_tools = inspirehep.modules.tools:INSPIRETools',
-            'invenio_records_rest = invenio_records_rest:InvenioRecordsREST',
+            'inspire_tools = inspirehep.modules.tools:INSPIRETools'
```
Comment (jacquerie) [inline setup.py]: As the dirty diff shows, missing comma.

File: inspirehep/modules/pidstore/utils.py [hunk 2]
```diff
@@ -24,20 +24,29 @@
...
 
+from flask import current_app
+
 from six.moves.urllib.parse import urlsplit
 
-from invenio_records_rest import current_records_rest
+
+def _get_pid_type_endpoint_map():
+    pid_type_endpoint_map = {}
+    for key, value in current_app.config['RECORDS_REST_ENDPOINTS'].iteritems():
```
Comment (jacquerie) [inline inspirehep/modules/pidstore/utils.py]: `iteritems()` is Python 2 only, better use `six.iteritems` here.

**Comment by jmartinm** [comment]
Tests pass with `invenio-db[postgresql,versioning]==1.0.0b2`. It seems like `test_permissions.py` has problems with `invenio-db==1.0.0b3`. To be further investigated.

**Comment by jacquerie** [comment]
This guy should be split in two commits, one fixing the build, one fixing the bug. Can I do it?

**Comment by jmartinm** [comment]
Sure, please go ahead. I can check tomorrow how to fix the `test_permissions.py` with `b3`

**Pull request closed without merging**